nemo/core/classes/modelPT.py

```diff
@@ -110,6 +112,9 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
 
         self._cfg = cfg
 
+        # init mapping submodule attribute -> config_field for nested NeMo models
+        self.nemo_submodule_name_to_config_field = dict()
```

Comment:
```suggestion
        self._nemo_submodule_name_to_config_field = dict()
```

Can we protect this?

Reply:
Yep, I agree. Fixed 👍